.goreleaser.yaml

```diff
@@ -118,4 +118,4 @@ changelog:
 release:
   prerelease: auto
 git:
-  prerelease_suffix: "-rc"
+  prerelease_suffix: "-*"
```

Comment:
Are you sure wildcard like `*` works here? It didn't work before. See https://github.com/celestiaorg/celestia-app/pull/3833

I think we may want to use this config:

```
# .goreleaser.yaml
git:
  # What should be used to sort tags when gathering the current and previous
  # tags if there are more than one tag in the same commit.
  #
  # Default: '-version:refname'.
  tag_sort: -version:creatordate

  # What should be used to specify prerelease suffix while sorting tags when gathering
  # the current and previous tags if there are more than one tag in the same commit.
  prerelease_suffix: "-"
```

See https://goreleaser.com/customization/git/?h=prerelease_suffix

Reply:
Yes, I did some test and everything is working fine.

See https://github.com/pippokr/celestia-app/releases/tag/v2.2.7-arabica

Reply:
Thanks for testing! Do you mind creating a few more tags to verify this works in all edge cases. For example: can you please create:
1. v2.3.0-rc0
2. v2.3.0-arabica
3. v2.3.0-mocha
4. v2.3.0

The GoReleaser CI for each invocation should use fetch the correct tag and attach binaries to the correct release. Also all releases should be considered prerelease except for the last one

Reply:
No bother at all :)

I've created some additional tags as you requested and as you can see, the binaries have been created and attached to the correct release.
https://github.com/pippokr/celestia-app/releases

Reply:
Confirmed it seems to work:

v2.3.0-rc0: https://github.com/pippokr/celestia-app/actions/runs/11158938550/job/31016375536#step:6:93
v2.3.0-arabica: https://github.com/pippokr/celestia-app/actions/runs/11161716747/job/31024948289#step:6:93
v2.3.0-mocha: https://github.com/pippokr/celestia-app/actions/runs/11161986186/job/31025762310#step:6:92
v2.3.0: https://github.com/pippokr/celestia-app/actions/runs/11162196323/job/31026407515#step:6:93

Reply:
Happy days :)